enterprise/frontend/src/embedding-sdk/components/private/SdkThemeProvider.tsx

```diff
@@ -32,13 +33,15 @@ export const SdkThemeProvider = ({ theme, children }: Props) => {
 
   return (
     <ThemeProvider theme={themeOverride}>
-      <GlobalStyles />
+      <SDKGlobalStyles />
       {children}
     </ThemeProvider>
   );
 };
 
-function GlobalStyles() {
+function SDKGlobalStyles() {
```

Comment:
I renamed it because we have another component `GlobalStyles`, and I spent quite some time being confused because the other one actually sets the font in another way

Reply:
This is a great change 👍